plugins/vendor/zoom/lib/model/kZoomEventHanlder.php

```diff
@@ -306,12 +307,13 @@ protected static function loadDropFolderFiles($dropFolderId)
 		return $dropFolderFilesMap;
 	}
 	
-	protected static function retrieveByFolderIdOrderAndStatusesNotIn($dropFolderId, $order, $statuses)
+	protected static function retrieveByFolderIdOrderAndStatusesNotIn($dropFolderId, $order, $statuses, $fileName)
 	{
 		$c = new Criteria();
 		$c->addAnd(DropFolderFilePeer::DROP_FOLDER_ID, $dropFolderId, Criteria::EQUAL);
 		$c->addAnd(DropFolderFilePeer::STATUS, $statuses, Criteria::NOT_IN);
 		$c->addAnd(DropFolderFilePeer::CREATED_AT, time() - dateUtils::DAY * 3, Criteria::GREATER_EQUAL);
+		$c->addAnd(DropFolderFilePeer::FILE_NAME, $fileName, Criteria::EQUAL);
 		$c->addAscendingOrderByColumn($order);
```

Comment:
ordering is less relevant and we can remove it from the function as we will have only one result from now on